core/controllers/concept_card_viewer.py

```diff
@@ -15,26 +15,26 @@
 """Controllers for the Oppia skill's concept card viewer."""
 
 from core.controllers import base
+from core.domain import acl_decorators
 from core.domain import skill_services
 import feconf
 
 
 class ConceptCardDataHandler(base.BaseHandler):
     """A card that shows the explanation of a skill's concept."""
 
+    @acl_decorators.can_play_skill
```

Comment:
I don't know what "can play skill" means. Probably OK for this to be the same as can_play_explorations or even open access, but "can view skill" is OK too I think.

Reply:
I tried having no decorator, but threw the backend error last time, as every method required a decorator, so added this. If it is to be can_play_exploration, then should I be passing exploration id also?

Reply:
Oh yeah good point, sorry. It's probably not can_play_exploration then. Maybe go with either "can view skill" or "open access".